app/src/test/java/org/simple/clinic/settings/changelanguage/ChangeLanguageEffectHandlerTest.kt

```diff
@@ -89,14 +94,24 @@ class ChangeLanguageEffectHandlerTest {
 
   @Test
   fun `when the restart activity effect is received, the restart activity ui action must be invoked`() {
-    // given
-    testCase.dispatch(RestartActivity)
-
     // when
-    testCase.assertNoOutgoingEvents()
+    testCase.dispatch(RestartActivity)
```

Comment:
Whoops, I didn't realise this was incorrectly formatted. Thanks for fixing this! 🚀

Reply:
Yeah, I just copied the above code and didn't realise it was not formatted correctly 😅